### ISSUE TITLE
Username field for accessibility

### ISSUE THREAD

**Issue by criverosv** [criverosv/PA-semana4#1]
**Describe the bug**

Password forms should have (optionally hidden) username fields for accessibility

**To Reproduce**

1. Go to the Staff page http://localhost:2368/ghost/#/staff/
2. Click on a staff name

**Expected behavior**

No error should appear on the console

**Screenshots**
![103](https://user-images.githubusercontent.com/79058155/116830530-7d314580-ab70-11eb-9af6-077d954d3894.png)

**Desktop (please complete the following information):**
 - Chronium
